_posts/2015-06-20-mental-number-line.md

```diff
@@ -0,0 +1,47 @@
+Studies have found this mental number line to be pervasive across tasks where people judge size, duration or emotional expression (preferring frownies on the left and smileys on the right, for example).
+But is the mental number line just about number or is it part of a more general system that extends to any judgment of magnitude?
+
+Researchers sought to test this using a standard experiment with the addition of an irrelevant magnitude, a weight worn on the wrist.
+Twenty-four participants had to quickly and accurately judge whether a number (0-9) on a screen is odd or even.
+They gave their responses using a keyboard, and the experiment was balanced so that the 'odd' and 'even' responses were not always given with keys on the left ('Q') or right ('P') side.
+There were three conditions: participants wore a five-pound weight on the left wrist, the right wrist, or no weight (the baseline condition).
+
+![Young chicks use a mental number line that reads left to right, just like humans]({{ site.url }}/images/2015-06-20-rugani-chick.jpg)
+
+> _Young chicks use a mental number line that reads left to right, just like humans. [Image](http://media.eurekalert.org/scipak/gallery/images/2015-01/rugani1HR.jpg) by Rosa Rugani, University of Padova_
+
+According to the SNARC [(Spatial-Numerical Association of Response Codes)](https://en.wikipedia.org/wiki/Spatial-numerical_association_of_response_codes) effect usually seen in this experiment, people should react more quickly when stimuli conform to the mental number line.
+That is, small numbers should elicit a faster left response and large numbers a faster right response.
+The right-weighted condition should be similar to the baseline condition, because the increased weight on the right wrist agrees with the mental number line idea of greater magnitude being associated with a spatial location on the right.
+The left-weighted condition is in conflict with the association of smaller numbers (or magnitudes) with the left side of space.
+If the mental number line is specific to number, however, an external factor like weight shouldn't affect performance.
+
+First, the researchers removed any data with wrong answers or very slow responses.
+On average, people responded equally fast in all three conditions.
+In comparing the left and right responses, the authors found similar results for the baseline and right conditions: the number line effect was present, with left and right side responses faster for smaller and larger numbers, respectively.
+The left condition, however, did not show this effect.
+With left-weighted wrists, participants did not show any responses time differences in a left-right direction.
+There are weren't any differences across conditions depending on which hand participants had used in their responses and, although most of the participants were right-handed, handedness also did not affect the results.
+
```

Comment:
I feel like this paragraph could've been explained a lot clearer -- I'm not saying there's anything wrong with it, but the repeated "left"/"right" stuff made my head spin a bit. :P